practices/risk_analysis.md

```diff
@@ -0,0 +1,76 @@
+---
+slug: risk-analysis
+stages:
+  - development
+short_description: Risk Analysis and Management is a key project management practice to ensure that the least number of surprises occur while the project is underway
+tags:
+  - risk analysis
+  - software documentation
+keywords:
+  - risk analysis
+  - documentation
+  - task
+---
+
+# Risk Analysis
+
+**TL;DR**
+
+Risk analysis is the process of identifying and analyzing potential issues that could negatively impact key business initiatives or projects. This process is done in order to help organizations avoid or mitigate those risks.
+
+## What Is Risk Analysis
+
+Risk analysis is the process of analyzing, determining, and defining the risk of danger to government agencies, and businesses. We also use it to determine risk to individuals, towns, cities, and whole national economies.
+
+Specifically, the analysis looks at potential natural and anthropogenic events. Anthropogenic events are events that human activity causes.
+
+In IT, we commonly use a risk analysis report to align technological goals with those of the company. IT stands for information technology. Risk analysis is both quantitative or qualitative.
+
+![Process](/files/risk_analysis.jpg)
+
+## Why You Might Want the Risk Analysis
+1. If you complete preventative risk analysis you can potentially avoid making huge losses
+2. Risk analysis will help your business avoid security breaches
+3. If you fail to plan for future success, you plan to fail
+4. You’ll be able to draft up new policies to prevent potential risks from eventuating
+5. By performing risk analysis you may be able to increase your business’ profits
+
+
+## How to Implement the Risk Analysis
+**1. Identify Threats**
+
+The first step in Risk Analysis is to identify the existing and possible threats that you might face. These can come from many different sources. For instance, they could be:
+
+
+- Human – Illness, death, injury, or other loss of a key individual.
+- Operational – Disruption to supplies and operations, loss of access to essential assets, or failures in distribution.
+- Reputational – Loss of customer or employee confidence, or damage to market reputation.
+- Procedural – Failures of accountability, internal systems, or controls, or from fraud.
+- Project – Going over budget, taking too long on key tasks, or experiencing issues with product or service quality.
+- Financial – Business failure, stock market fluctuations, interest rate changes, or non-availability of funding.
+- Technical – Advances in technology, or from technical failure.
+- Natural – Weather, natural disasters, or disease.
+- Political – Changes in tax, public opinion, government policy, or foreign influence.
+
+Tools such as **SWOT Analysis** ,**Failure Mode** and **Effects Analysis** can also help you uncover threats, while **Scenario Analysis** helps you explore possible future threats.
+
+**2. Estimate Risk**
```

Comment:
Please, rewrite this part in your own words, it seems that there is heavy connection with _2. Estimate Risk_

_Once you’ve identified the threats you’re facing, you need to calculate out both the likelihood of these threats being realized, and their possible impact._

_One way of doing this is to make your best estimate of the probability of the event occurring, and then to multiply this by the amount it will cost you to set things right if it happens. This gives you a value for the risk:_
_Risk Value = Probability of Event x Cost of Event_

Reply:
done